README.md

```diff
@@ -428,7 +428,7 @@ contains_substring_dcba = DFA.contains_subsequence({'a', 'b', 'c', 'd'}, 'dcba')
 avoids_substring_dcba = DFA.contains_subsequence({'a', 'b', 'c', 'd'}, 'dcba', contains=False)
 ```
 
-#### DFA.of_length(cls, input_symbols, min_length=0, max_length=None)
+#### DFA.of_length(cls, input_symbols, min_length=0, max_length=None, symbols_to_count=None)
```

Comment:
@eliotwrobson Can you please add documentation for the purpose of `symbols_to_count`?

Reply:
Done, and added documentation to the `count_mod` for `symbols_to_count` that was not present.